diff --git a/client/static/xsd/cnxml.xsd b/client/static/xsd/cnxml.xsd
index 6091f11..04071d3 100644
--- a/client/static/xsd/cnxml.xsd
+++ b/client/static/xsd/cnxml.xsd
@@ -76,36 +76,13 @@
   </xs:group>
   <xs:element name="metadata">
     <xs:complexType>
-      <xs:choice minOccurs="0" maxOccurs="unbounded">
-        <xs:group ref="mathml:content-id_2"/>
-        <xs:group ref="mathml:repository_2"/>
-        <xs:group ref="mathml:content-url_2"/>
-        <xs:group ref="mathml:title_3"/>
-        <xs:group ref="mathml:short-title_2"/>
-        <xs:group ref="mathml:subtitle_2"/>
-        <xs:group ref="mathml:version_2"/>
-        <xs:group ref="mathml:created_2"/>
-        <xs:group ref="mathml:revised_2"/>
-        <xs:group ref="mathml:actors_2"/>
-        <xs:group ref="mathml:roles_2"/>
-        <xs:group ref="mathml:license_2"/>
-        <xs:group ref="mathml:extended-attribution_2"/>
-        <xs:group ref="mathml:derived-from_2"/>
-        <xs:group ref="mathml:keywordlist_2"/>
-        <xs:group ref="mathml:subjectlist_2"/>
-        <xs:group ref="mathml:education-levellist_2"/>
-        <xs:group ref="mathml:abstract_2"/>
-        <xs:group ref="mathml:objectives_2"/>
-        <xs:group ref="mathml:homepage_2"/>
-        <xs:group ref="mathml:institution_2"/>
-        <xs:group ref="mathml:course-code_2"/>
-        <xs:group ref="mathml:instructor_2"/>
-        <xs:group ref="mathml:uuid_2"/>
-        <xs:group ref="mathml:canonical-book-uuid_2"/>
-        <xs:group ref="mathml:slug_2"/>
-        <xs:group ref="mathml:language_2"/>
-        <xs:element ref="mdml:super"/>
-      </xs:choice>
+      <xs:all>
+        <xs:element ref="mdml:content-id"/>
+        <xs:element minOccurs="0" ref="mdml:title"/>
+        <xs:element ref="mdml:uuid"/>
+        <xs:element minOccurs="0" ref="mdml:abstract"/>
+        <xs:element minOccurs="0" ref="mdml:super"/>
+      </xs:all>
       <xs:attribute ref="cmlnle:case"/>
       <xs:attribute ref="cmlnle:reference"/>
       <xs:attribute ref="cxlxt:born"/>
diff --git a/client/static/xsd/collxml.xsd b/client/static/xsd/collxml.xsd
index c8d2dfc..32589fe 100644
--- a/client/static/xsd/collxml.xsd
+++ b/client/static/xsd/collxml.xsd
@@ -29,37 +29,14 @@
   </xs:element>
   <xs:element name="metadata">
     <xs:complexType>
-      <xs:choice minOccurs="0" maxOccurs="unbounded">
-        <xs:group ref="mathml:content-id"/>
-        <xs:group ref="mathml:repository"/>
-        <xs:group ref="mathml:content-url"/>
-        <xs:group ref="mathml:title_2"/>
-        <xs:group ref="mathml:short-title"/>
-        <xs:group ref="mathml:subtitle"/>
-        <xs:group ref="mathml:version"/>
-        <xs:group ref="mathml:created"/>
-        <xs:group ref="mathml:revised"/>
-        <xs:group ref="mathml:actors"/>
-        <xs:group ref="mathml:roles"/>
-        <xs:group ref="mathml:license"/>
-        <xs:group ref="mathml:extended-attribution"/>
-        <xs:group ref="mathml:derived-from"/>
-        <xs:group ref="mathml:keywordlist"/>
-        <xs:group ref="mathml:subjectlist"/>
-        <xs:group ref="mathml:education-levellist"/>
-        <xs:group ref="mathml:abstract"/>
-        <xs:group ref="mathml:objectives"/>
-        <xs:group ref="mathml:homepage"/>
-        <xs:group ref="mathml:institution"/>
-        <xs:group ref="mathml:course-code"/>
-        <xs:group ref="mathml:instructor"/>
-        <xs:group ref="mathml:uuid"/>
-        <xs:group ref="mathml:canonical-book-uuid"/>
-        <xs:group ref="mathml:slug"/>
-        <xs:element ref="mdml:ancillary"/>
-        <xs:element ref="mdml:version-history"/>
-        <xs:group ref="mathml:language"/>
-      </xs:choice>
+      <xs:all>
+        <xs:element minOccurs="0" ref="mdml:content-id"/>
+        <xs:element ref="mdml:title"/>
+        <xs:element ref="mdml:license"/>
+        <xs:element ref="mdml:uuid"/>
+        <xs:element ref="mdml:slug"/>
+        <xs:element ref="mdml:language"/>
+      </xs:all>
       <xs:attribute ref="xml:lang"/>
       <xs:attribute name="class" type="xs:token"/>
       <xs:attribute ref="s:read-only"/>
diff --git a/client/static/xsd/mathml.xsd b/client/static/xsd/mathml.xsd
index 628bd82..48f2bff 100644
--- a/client/static/xsd/mathml.xsd
+++ b/client/static/xsd/mathml.xsd
@@ -3086,6 +3086,36 @@
               <xs:element ref="mathml:pi"/>
               <xs:element ref="mathml:eulergamma"/>
               <xs:element ref="mathml:infinity"/>
+              <xs:group ref="mathml:mi"/>
+              <xs:group ref="mathml:mn"/>
+              <xs:group ref="mathml:mo"/>
+              <xs:group ref="mathml:mtext"/>
+              <xs:element ref="mathml:mspace"/>
+              <xs:group ref="mathml:ms"/>
+              <xs:element ref="mathml:maligngroup"/>
+              <xs:element ref="mathml:malignmark"/>
+              <xs:group ref="mathml:mrow"/>
+              <xs:group ref="mathml:mfrac"/>
+              <xs:group ref="mathml:msqrt"/>
+              <xs:group ref="mathml:mroot"/>
+              <xs:group ref="mathml:mstyle"/>
+              <xs:group ref="mathml:merror"/>
+              <xs:group ref="mathml:mpadded"/>
+              <xs:group ref="mathml:mphantom"/>
+              <xs:group ref="mathml:mfenced"/>
+              <xs:group ref="mathml:menclose"/>
+              <xs:group ref="mathml:msub"/>
+              <xs:group ref="mathml:msup"/>
+              <xs:group ref="mathml:msubsup"/>
+              <xs:group ref="mathml:munder"/>
+              <xs:group ref="mathml:mover"/>
+              <xs:group ref="mathml:munderover"/>
+              <xs:group ref="mathml:mmultiscripts"/>
+              <xs:group ref="mathml:mtable"/>
+              <xs:group ref="mathml:mstack"/>
+              <xs:group ref="mathml:mlongdiv"/>
+              <xs:group ref="mathml:maction"/>
+              <xs:group ref="mathml:semantics_2"/>
             </xs:choice>
             <xs:choice minOccurs="0" maxOccurs="unbounded">
               <xs:element ref="mathml:annotation"/>
@@ -17988,6 +18018,36 @@
               <xs:element ref="mathml:pi"/>
               <xs:element ref="mathml:eulergamma"/>
               <xs:element ref="mathml:infinity"/>
+              <xs:group ref="mathml:mi"/>
+              <xs:group ref="mathml:mn"/>
+              <xs:group ref="mathml:mo"/>
+              <xs:group ref="mathml:mtext"/>
+              <xs:element ref="mathml:mspace"/>
+              <xs:group ref="mathml:ms"/>
+              <xs:element ref="mathml:maligngroup"/>
+              <xs:element ref="mathml:malignmark"/>
+              <xs:group ref="mathml:mrow"/>
+              <xs:group ref="mathml:mfrac"/>
+              <xs:group ref="mathml:msqrt"/>
+              <xs:group ref="mathml:mroot"/>
+              <xs:group ref="mathml:mstyle"/>
+              <xs:group ref="mathml:merror"/>
+              <xs:group ref="mathml:mpadded"/>
+              <xs:group ref="mathml:mphantom"/>
+              <xs:group ref="mathml:mfenced"/>
+              <xs:group ref="mathml:menclose"/>
+              <xs:group ref="mathml:msub"/>
+              <xs:group ref="mathml:msup"/>
+              <xs:group ref="mathml:msubsup"/>
+              <xs:group ref="mathml:munder"/>
+              <xs:group ref="mathml:mover"/>
+              <xs:group ref="mathml:munderover"/>
+              <xs:group ref="mathml:mmultiscripts"/>
+              <xs:group ref="mathml:mtable"/>
+              <xs:group ref="mathml:mstack"/>
+              <xs:group ref="mathml:mlongdiv"/>
+              <xs:group ref="mathml:maction"/>
+              <xs:group ref="mathml:semantics_2"/>
             </xs:choice>
             <xs:choice minOccurs="0" maxOccurs="unbounded">
               <xs:element ref="mathml:annotation"/>
diff --git a/client/static/xsd/mdml.xsd b/client/static/xsd/mdml.xsd
index 8bf37cc..b745ccc 100644
--- a/client/static/xsd/mdml.xsd
+++ b/client/static/xsd/mdml.xsd
@@ -10,6 +10,41 @@
   <xs:import namespace="http://www.w3.org/1998/Math/MathML" schemaLocation="mathml.xsd"/>
   <xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="xml.xsd"/>
   <xs:import namespace="urn:oasis:names:tc:opendocument:xmlns:xsl-fo-compatible:1.0" schemaLocation="ns1.xsd"/>
+  <xs:element name="content-id" type="xs:string"/>
+  <xs:element name="title" type="xs:string"/>
+  <xs:element name="language" type="xs:string"/>
+  <xs:element name="uuid" type="xs:string"/>
+  <xs:element name="slug" type="xs:string"/>
+  <xs:element name="abstract">
+    <xs:complexType mixed="true">
+      <xs:choice maxOccurs="unbounded">
+        <xs:group ref="mathml:para_2"/>
+        <xs:choice minOccurs="0" maxOccurs="unbounded">
+          <xs:group ref="mathml:emphasis"/>
+          <xs:group ref="mathml:term"/>
+          <xs:group ref="mathml:foreign"/>
+          <xs:group ref="mathml:cite"/>
+          <xs:group ref="mathml:span"/>
+          <xs:group ref="mathml:sup"/>
+          <xs:group ref="mathml:sub"/>
+          <xs:group ref="mathml:code_2"/>
+          <xs:group ref="mathml:math"/>
+          <xs:group ref="mathml:quote_2"/>
+          <xs:group ref="mathml:preformat_2"/>
+          <xs:group ref="mathml:list_2"/>
+        </xs:choice>
+      </xs:choice>
+      <xs:attribute ref="xml:lang"/>
+      <xs:attribute name="class" type="xs:token"/>
+      <xs:attribute ref="s:read-only"/>
+      <xs:anyAttribute processContents="skip"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="license">
+    <xs:complexType mixed="true">
+      <xs:attribute name="url" use="required"/>
+    </xs:complexType>
+  </xs:element>
   <xs:group name="content-id">
     <xs:sequence>
       <xs:element name="content-id">
